example/quick_starter.md

```diff
@@ -0,0 +1,155 @@
+
+<div align="center">
+
+![](./image/quick_starter_task.jpeg)
+
+</div>
+
+In `tasks.py`, `TaskC`, `TaskD`, and `TaskD` inherits `kannon.TaskOnBullet` instead of `gokart.TaskOnKart`, so Task C0-C2, D0-D2, and E0-E2 are run on different jobs in parallel.
+
+```python
+# example/tasks.py
+class TaskC(kannon.TaskOnBullet):
+    ...
+
+class TaskD(kannon.TaskOnBullet):
+    ...
+
+class TaskE(kannon.TaskOnBullet):
+    ...
+```
+
+Since Task C, D, and E are tasks that will take longer time than other tasks and they can be run in parallel, `kannon` will be very effective! Let's check it.
+
+
+### 5. Run master job
+Run the master job!
+```bash
+$ minikube kubectl -- apply -f k8s/master-job.yaml
+job.batch/master-job-quick-starter created
+```
+
+Let's check running jobs. You can see 3 child tasks are running on multiple jobs in parallel.
+```bash
+$ minikube kubectl -- get jobs -n kannon-quick-starter
+NAME                                     COMPLETIONS   DURATION   AGE
+master-job-quick-starter                 0/1           7m23s      7m23s
+# task c0, c1, c2 are done in parallel.
+quick-starter-taskc-014-20230413142312   1/1           3m5s       6m52s
+quick-starter-taskc-036-20230413142309   1/1           3m5s       6m55s
+quick-starter-taskc-191-20230413142306   1/1           3m6s       6m58s
+# then, task d0, d1, d2 are done in parallel.
+quick-starter-taskd-079-20230413142613   1/1           3m7s       3m51s
+quick-starter-taskd-118-20230413142615   1/1           3m6s       3m49s
+quick-starter-taskd-143-20230413142617   1/1           3m7s       3m47s
+# finally, task e0, e1, e2 are running in parallel now.
+quick-starter-taske-071-20230413142923   0/1           41s        41s
+quick-starter-taske-091-20230413142924   0/1           40s        40s
+quick-starter-taske-146-20230413142922   0/1           42s        42s
+```
+
+### 6. Check the result
+
+```bash
+$ minikube kubectl --  get jobs -n kannon-quick-starter
+NAME                                     COMPLETIONS   DURATION   AGE
+master-job-quick-starter                 1/1           11m        14m
+quick-starter-taskc-014-20230413142312   1/1           3m5s       14m
+quick-starter-taskc-036-20230413142309   1/1           3m5s       14m
+quick-starter-taskc-191-20230413142306   1/1           3m6s       14m
+quick-starter-taskd-079-20230413142613   1/1           3m7s       11m
+quick-starter-taskd-118-20230413142615   1/1           3m6s       11m
+quick-starter-taskd-143-20230413142617   1/1           3m7s       11m
+quick-starter-taske-071-20230413142923   1/1           4m5s       8m12s
+quick-starter-taske-091-20230413142924   1/1           4m6s       8m11s
+quick-starter-taske-146-20230413142922   1/1           4m5s       8m13s
+```
+
+All jobs are completed successfully!
+Duration for the master job to be completed is `11m`.
+Compared to the case without `kannon`, it is almost **3x** faster!
+
+|using kannon? | duration |
+|---|---|
+|yes | **11 minutes**|
+|no | 31 minutes|
```

Comment:
[imo]

Don't you reduce sleep time?

I tried to follow this README and felt that waiting 11min is too long, especially because there's no any progress feedback.

Reply:
It is indeed long. 
Now the clock period is 1 sec, so I set the tutorial's job duration to 10~min to make you appreciate the parallelization.
How about setting the clock period to 0.1sec so that it all finishes in 2-3min?

A separate PR will be issued to change the clock period from 1 sec to 0.1 sec.

Reply:
Reduced to 2-3min!